features/url-rewriters/imgix.md

```diff
@@ -0,0 +1,37 @@
+# Imgix URL-rewriter
+
+ImageKit can translate transformations written in Imgix's syntax to ImageKit's syntax. Learn how to use a URL-rewriter on [this page](/features/url-rewriters).
+
+{% hint style="info" %}
+**Beta feature**\
+This feature is currently in beta. Some transformations may not work as expected.
+{% endhint %}
+
+## Supported Transformations
+
+The following Imgix transformations are supported for translation:
+
+| Imgix transform | Example translation | Remarks |
+| - | - | - |
+| <p>width</p> | <ul> <li> `w=100` => `w-100`</li> </ul> |  |
+| <p>height</p> | <ul> <li>`h=100` => `h-100`</li> </ul> |  |
+| <p>aspect ratio</p> | <ul> <li>`ar=2:3` => `ar-2-3`</li><li>`ar=3:1` => `ar=3-1`</li><li>`ar=2:4` => `ar=2-4`</li> </ul> | <ul>Aspect ratio will only work when we  pass either width or height (not both) and fit</ul> |
+| <p>crop</p> | <ul> <li>`crop=top` => `fo-top`</li><li>`crop=top,left` => `fo-top_left`</li><li>`crop=left,top` => `fo-top_left`</li><li>`crop=faces` => `fo-auto`</li> </ul> | <ul>Supported values: <ul><li>top</li><li>left</li><li>right</li><li>bottom</li><li>top,left</li><li>top,right</li><li>bottom,right</li><li>bottom,left</li><li>left,top</li><li>right,top</li><li>right,bottom</li><li>left,bottom</li><li>faces</li></ul></ul> |
+| <p>fit</p> | <ul> <li>`fit=clip` => `c-at_max_enlarge`</li><li>`fit=fill` => `cm-pad_resize`</li><li>`fit=scale` => `c-force`</li> </ul> | <ul>Supported values: <ul><li>clip => c-at_max_enlarge </li><li>crop => no mapping required</li><li>fill => cm-pad_resize</li><li>fillmax => cm-pad_resize </li><li>max => c_at_max </li><li>scale => c-force</li><li>facearea => fo-face</li></ul></ul><ul>For fit=fillmax when transformed dimensions are more than original dimensions then padding will only take place either along h or w not both </ul> |
```

Comment:
- Write all the mappings in the second column of the table in the format "fit-xyz => c/cm-abc". In the third column, only write a list of all possible fit values, not their mappings.
- Replace the note with "For fit=fillmax, if the requested dimensions are greater than the original dimensions, then padding will be added along either one of the dimensions, not both"